lib/helpers.bash

```diff
@@ -243,6 +243,8 @@ _disable-thing ()
         rm $BASH_IT/$subdirectory/enabled/$(basename $plugin)
     fi
 
+    exec ${0/-/}
```

Comment:
It's like a more thorough reload... Sourcing .bashrc/.bash_profile is an additive process in that it only adds new things or redefines things that already existed before, since it's still the same process... If you want to disable or remove anything, you've got to actually restart your session...

Running 'exec bash' kills the current shell process and replaces it with a new one (but is transparent to the user), so it gets around the need to have to manually restart a session when disabling things.